src/uuid.cr

```diff
@@ -164,6 +164,13 @@ struct UUID
     to_slice == other.to_slice
   end
 
+  # Convert to `String` in literal format.
+  def inspect(io : IO)
+    io << "\""
```

Comment:
I personally think these should be removed per [this comment](https://github.com/crystal-lang/crystal/issues/5552#issuecomment-355815882). But please wait for a consensus before actually changing this.

Reply:
I agree. But if it is decided otherwise, you could use `"`'' (here and two lines down) and `%("#{subject}")` (in the spec) to avoid escaping double quotes.

Reply:
So what is the consensus on this issue?

Reply:
Remove.